quicktile.py

```diff
@@ -581,6 +581,9 @@ def get_workarea(self, monitor, ignore_struts=False):
             for g in struts:
                 # http://standards.freedesktop.org/wm-spec/1.5/ar01s05.html
                 # XXX: Must not cache unless watching for notify events.
+
+		# a hack to eliminate the offset from the top bar
```

Comment:
As-is, it looks like this comment applies to the `_Su` lines as well. Is there room to put it on the same line as the `g[2] = 0` without going over the PEP8 80-column limit?